ratis-netty/src/test/java/org/apache/ratis/netty/MiniRaftClusterWithNetty.java

```diff
@@ -59,6 +74,7 @@ protected RaftServerProxy newRaftServer(
       RaftPeerId id, StateMachine.Registry stateMachineRegistry , RaftGroup group,
       RaftProperties properties) throws IOException {
     NettyConfigKeys.Server.setPort(properties, getPort(id, group));
+    NettyConfigKeys.DataStream.setPort(properties, getDataStreamPort(id, group));
```

Comment:
What's the relationship between this propertis and the propertis in the following code, why need setPort twice?
![image](https://user-images.githubusercontent.com/51938049/98908844-fcb45500-24fb-11eb-8be3-24957f4beb7b.png)

Reply:
In `TestDataStreamNetty`, MiniRaftCluster is not used. This line is setup the port for MiniRaftCluster. 

I am asking above for whether we want to merge current tests to only use MiniRaftCluster.